addons/mail/static/src/messaging.js

```diff
@@ -402,7 +403,7 @@ export class Messaging {
         const subtype = isNote ? "mail.mt_note" : "mail.mt_comment";
         const params = {
             post_data: {
-                body,
+                body: new MailMessage(body).body,
```

Comment:
You don't need an instance of class here.
It should just be a function call.
Internally the function might need some state, but it's internal stuff of function. Here you just need to make a prettier body from a raw body.